### PR TITLE
Replace addConstr() by newer addLConstr() available for gurobi >= 9.0

### PR DESCRIPTION
**Pull request by nhuet**
- this remove some warnings that pops up in pycharm
- this is supposed to be faster
- this has been introduced in gurobi 9
- we add the minimal version for gurobi in install documentation (this is an optional requirement)

### CODE REVIEW

File: discrete_optimization/rcpsp/solver/rcpsp_lp_solver.py [hunk 1]
```diff
@@ -666,7 +666,7 @@ def init_model(self, **args):
                 self.x[key] * key[2] for key in variable_per_task[last_task]
             )
         )
-        self.model.addConstrs(
+        self.model.addLConstrs(
```
Comment (g-poveda) [inline discrete_optimization/rcpsp/solver/rcpsp_lp_solver.py:669]: to reverse (addLConstrs doesn't exist)

File: discrete_optimization/rcpsp/solver/rcpsp_lp_solver.py [hunk 2]
```diff
@@ -685,7 +685,7 @@ def init_model(self, **args):
                 r: [non_renewable[r]] * len(self.index_time) for r in non_renewable
             }
 
-        self.model.addConstrs(
+        self.model.addLConstrs(
```
Comment (g-poveda) [inline discrete_optimization/rcpsp/solver/rcpsp_lp_solver.py:688]: here too (and so on)

**Review by g-poveda — CHANGES_REQUESTED** [review]
we have to keep addContr**s** as it was, and just replace the addConstr by addLConstr
otherwise it seems ok

**Comment by nhuet** [comment]
> we have to keep addContr**s** as it was, and just replace the addConstr by addLConstr otherwise it seems ok

Done. (but already squashed in first commit, sorry about that)

**Review by g-poveda — APPROVED** [review]

**Pull request merged**